histomicstk/annotations_and_masks/annotation_and_mask_utils.py

```diff
@@ -39,6 +39,96 @@ def get_image_from_htk_response(resp):
 # %%===========================================================================
 
 
+def scale_slide_annotations(slide_annotations, sf):
+    """Scales up or down annotations in a slide.
+
+    Works in place, but returns slide_annotations anyways.
+
+    Parameters
+    -----------
+    slide_annotations : list of dicts
+        response from server request
+
+    sf : float
+        scale factor to multiply coordinates
+
+    Returns
+    ---------
+    list of dicts
+
+    """
+    if sf == 1.0:
+        return slide_annotations
+
+    for annidx, ann in enumerate(slide_annotations):
+        for elementidx, element in enumerate(ann['annotation']['elements']):
+            for key in element.keys():
```

Comment:
This could be more compactly done by
```
for key, entry in element.items():
    if key in {'height', 'width'}:                    # Note that a set is marginally more efficient than a list
        element[key] = int(entry * sf)      
    elif key in {'center', 'points'}
        element[key] = (np.array(entry) * sf).astype(int).tolist()
```